.github/workflows/backend-skipped-checks.yml

```diff
@@ -4,12 +4,19 @@
 name: Backend
 
 on:
+  push:
+    branches:
+      - 'master'
+      - 'release-**'
+    paths:
+      - "docs/**"
+      - "**.md"
+      - "**/frontend/test/**"
```

Comment:
Maybe this is a dumb question but can't we skip backend tests for frontend source changes as well? Frontend tests obviously don't affect backend stuff but I don't see any reason why something like `NemanjasReactComponent.jsx` would affect them either.

Reply:
That is an excellent question! @camsaul 
Tbh, I wasn't sure how coupled some things are but yes - you are right. If all backend tests are pure unit tests, there is no reason why frontend changes should affect them.

We probably need another layer of contract testing, but until then... E2E tests will do.